code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -246,6 +250,11 @@ var/global/mulebot_count = 0
 			auto_pickup = !auto_pickup
 		if("report")
 			report_delivery = !report_delivery
+		if("ejectpai")

```

Comment:
It was there and everything, but this is the bit where I couldn't do the template compiling, and I finished this at like 7am.
I'll see if I can add it.